realm/src/main/java/io/realm/internal/Table.java

```diff
@@ -816,15 +829,26 @@ public void clearSubtable(long columnIndex, long rowIndex) {
      * @param index Index of row to fetch.
      * @return Unsafe row wrapper object.
      */
-    public UncheckedRow getUncheckedRow(long index) {
-        return UncheckedRow.get(context, this, index);
+    public UncheckedRow getUncheckedRowByIndex(long index) {
```

Comment:
This is a breaking change of an existing API. It is an internal API but it might be used in migrations.